US-CBP/cbp-theme
# Card expand

#### What's this PR do?
Allows cards to expand and contract

#### Where should the reviewer start?
Download repo and view demo-pages/card-archetype

#### How should this be manually tested?
View page and click the "MORE/LESS" button to see the card expand/contract and show tags.

#### Any background context you want to provide?
N/A
#### What are the relevant tickets?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
